heartbeat/server.js

```diff
@@ -175,24 +176,34 @@ function checkHeartbeat() {
             let flicDelayMillis = currentTime.diff(flicLastSeenTime)
             let heartbeatLastSeenTime = moment(doc.heartbeat_last_seen_time)
             let heartbeatDelayMillis = currentTime.diff(heartbeatLastSeenTime)
+            let pingLastSeenTime = moment(doc.flic_last_ping_time)
+            let pingDelayMillis = currentTime.diff(pingLastSeenTime)
         
             if(flicDelayMillis > FLIC_THRESHOLD_MILLIS && !doc.sent_alerts) {
                 log(`flic threshold exceeded; flic delay is ${flicDelayMillis} ms. sending alerts for ${doc.system_name}`)
                 updateSentAlerts(doc.system_id, true)
                 if(doc.muted) {
                     return
                 }
-                sendAlerts(doc.system_name)
+                sendAlerts('Flic', doc.system_name)
```

Comment:
This is ambiguous - can you change this to 'Darkstat' or 'Network monitor' or something?

Reply:
Done.